Automattic/pocket-casts-android
# filter_created Tracks event

## Description

Adds the filter_created event

## To Test

Just do this a few times with the different possible filter configurations:
1. Create a new filter
2. Observe that a `filter_created` event is fired with the appropriate properties for these keys:
    1. all_podcasts
    2. color
    3. downloaded
    4. duration
    5. episode_status_played
    6. episode_status_in_progress
    7. episode_status_unplayed
    8. icon_name
    9. media_type
    10. release_date
    11. starred

# Checklist
